tsconfig.json

```diff
@@ -0,0 +1,38 @@
+{
+  "compilerOptions": {
+    "target": "es6",                                /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */
+    "module": "commonjs",                           /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', 'es2020', or 'ESNext'. */
+    // "lib": [],                                   /* Specify library files to be included in the compilation. */
```

Comment:
I think that all commented options should be deleted. You can always keep it in you local copy and prepare a PR with them later

Reply:
> I think that all commented options should be deleted. You can always keep it in you local copy and prepare a PR with them later

🙄

this is unnecessary manufactured work which removes planning and intent from the codebase

Reply:
I mean, that you can do a PR with uncommented lines if there necessary in the future. Of course no need to commit commented lines

Reply:
I heard you, thanks.  I'm declining, because this is not a technical improvement, and you have developed a pattern of asking for changes of no value.

You requested five separate PRs here, four of which cannot exist and two of which are a single task.

You requested that I split a working build into something that cannot work and would take 20 minutes, which would involve checking an enormous number of intermediates into secondary storage, then receiving them again, hand-managing their identities, hitting storage limits, ***making the build not work outside of CI***, and there was literally no upside other than that you imagine somehow it was more proper

You want me to fix a security problem that Microsoft fixed three years ago that you can't name, don't understand correctly, and that never applied here in the first place

You're trying to prevent people who you imagine have the ability to check code in and publish from doing bad things, and you want this PR to wait until you've thought that through before it can land

***Please cut it out.  Thanks.***

All future comments should relate to specific technical improvements.  This is the third time I've said this to you.

Your preferences about comments have been noted.  I'm leaving these in.  Thank you for understanding